plush/string.pls

```diff
@@ -19,6 +19,20 @@ exports.toString = function (e)
     {
         return exports.intToString(e, 10);
     }
+    if (type == "array")
+    {
+        var builder = "[";
+		for (var i = 0;i < e.length;i += 1)
```

Comment:
@jammln Inconsistent use of spaces and tabs. Please change all tabs to 4 spaces. The indentation breaks on some editors because of mixed whitespaces.